editor/store/effects.js

```diff
@@ -499,4 +501,15 @@ export default {
 	SELECT_BLOCK: removeProvisionalBlock,
 
 	MULTI_SELECT: removeProvisionalBlock,
+
+	REMOVE_BLOCKS( action, { getState, dispatch } ) {
+		const state = getState();
+		const previousState = { ...state, editor: { present: last( state.editor.past ) } };
+		const firstRemovedBlockUID = action.uids[ 0 ];
+		const rootUID = getBlockRootUID( previousState, firstRemovedBlockUID );
+		const blockUIDToSelect = getPreviousBlockUid( previousState, firstRemovedBlockUID ) || rootUID;
```

Comment:
When will the fallback condition be met?

Reply:
When we are removing the first block nested in a column for example. When testing this case, I discovered a bug when removing a totally empty block inside a column this was caused because our code executed even when removing the provisional block, I added a check for this.